client/app/bundles/shared/components/Textarea.jsx

```diff
@@ -0,0 +1,91 @@
+//@flow
+import React from "react";
+import css from "./Textarea.scss";
+
+type Props = {
+  id?: string,
+  name?: string,
+  value?: string,
+  form?: string,
+  rows?: string | number,
+  cols?: string | number,
+  placeholder?: string,
+  label?: string,
+  autofocus?: boolean,
+  readonly?: boolean,
+  disabled?: boolean,
+  required?: boolean,
+  maxLength?: number
+};
+
+type State = {
+  value: string,
+  active: boolean
+};
+
+export default class Input extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = { value: "", active: false };
+  }
+
+  onChange = (e: SyntheticEvent<HTMLInputElement>) => {
+    e.preventDefault();
+    this.setState({ value: e.currentTarget.value });
+  };
+
+  onFocus = (e: SyntheticMouseEvent<HTMLInputElement>) => {
+    this.setState({ active: true });
```

Comment:
I don't think we need to pass in the event if we're not gonna use it.

Reply:
Thanks for point that out! Removed!